aspnet-core/.gitignore

```diff
@@ -250,3 +250,9 @@ paket-files/
 # JetBrains Rider
 .idea/
 *.sln.iml
+
+# NCrunch
```

Comment:
Oh, and I added NCrunch to the .gitignore.  It's not technically related to the PR, and I'm happy to separate it out if you like.